<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
 Released as open source by Gabriel Caudrelier

 Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com

 https://github.com/metrodango/pip3line

 Released under BSD 3-Clause see LICENSE for more information
 </comment>
 <class>BytesToFloatWdiget</class>
 <widget class="QWidget" name="BytesToFloatWdiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="endianGroupBox">
     <property name="title">
      <string>Endianness</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="bigEndianRadioButton">
        <property name="text">
         <string>Big endian</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="littleEndianRadioButton">
        <property name="text">
         <string>Little endian</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sizeGroupBox">
     <property name="title">
      <string>Float size</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="float64radioButton">
        <property name="text">
         <string>64 bits (double)</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="float32radioButton">
        <property name="text">
         <string>32 bits (float)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="precisionLabel">
        <property name="text">
         <string>Precision</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="precisionSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
